vivisect/analysis/generic/codeblocks.py

```diff
@@ -130,7 +130,8 @@ def analyzeFunction(vw, funcva):
         # So we don't add a codeblock if we're re-analyzing a function
         # (like during dynamic branch analysis)
         bsize = blocks[bva]
-        if bva not in oldblocks:
+        tmpcb = vw.getCodeBlock(bva)
+        if bva not in oldblocks or tmpcb is None:   # sometimes codeblocks can be deleted if owned by multiple functions
```

Comment:
That comment seems like something we should fix (in a different PR).

I remember at one point desiring to switch over our codeblock analysis stuff to using the FuncBlockGraph for this very reason, but decided to hold off due to prioritization.

Reply:
But overall nothing crazy. But do you have a unit test for this :smile:

Reply:
sorry, missed this comment.  no unit test for this.  it's something that bit me on a non-public binary a while back, and i no longer remember the exact context (which binary or what i did to cause this).  sorry.